### DISCUSSION TITLE
how to change colours of thick, mohawk styles

### DISCUSSION THREAD

**Discussion by fk20268** [dapilab/react-nice-avatar#31] [Q&A] [answered]
Hi, I'm using your package to build an avatar editor and I noticed that I can't change the colours of the thick or mohawk styles from black (I am able to with the other styles) , do you know how I would tackle this?

**Comment by wwayne** [comment, marked as answer]
Yeah you can add `hairColorRandom` (default is false) to your component so both of them can be set as other colors
